src/domains/movie/services/index.js

```diff
@@ -0,0 +1,34 @@
+const movieApi = (url, options) => {
+  const _options = {
+    ...options,
+    headers: {
+      Authorization: `Bearer ${import.meta.env.VITE_ACCESS_TOKEN}`,
+      accept: 'application/json',
+    },
+  };
+  const _url = 'https://api.themoviedb.org' + url;
+
+  return fetch(_url, _options);
+};
+
+export const getPopularMovie = async (params) => {
+  const { page } = params;
+
+  try {
+    const response = await movieApi(
+      `/3/discover/movie?include_adult=false&include_video=false&language=ko&region=kr&sort_by=popularity.desc&page=${page ?? 1}`,
+      { method: 'GET' },
+    );
+    const data = await response.json();
+
+    if (!response.ok) {
+      if (data.status_code === 22) {
+        throw new Error('현재 페이지를 찾을 수 없습니다.');
+      }
+    }
```

Comment:
22 가 어떤 응답인지 상수화 해두면 더욱 좋을 것 같아요.
또한 !response.ok에 대한 기본 에러 메시지가 없는데, 이 또한 고려되면 좋을 것 같네요!

```suggestion
    const NOT_FOUND_ERROR = 22;
    
    if (!response.ok) {
      if (data.status_code === NOT_FOUND_ERROR) {
        throw new Error('현재 페이지를 찾을 수 없습니다.');
      } else {
        throw new Error(data.status_message || '영화 데이터를 불러오는데 실패했습니다.');
      }
    }
```